backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/ZipApi.java

```diff
@@ -148,6 +145,20 @@ static String getZipTable(Request request, Response response) throws IOException
 
   static String getZippedReports(Request request, Response response) throws IOException {
     String reports = request.queryParams("id");
+    Map<String, Object> parameters = new LinkedHashMap<>();
+    request
+        .queryParams()
+        .forEach(
+            param -> {
+              if ("id".equals(param)) {
+                // ignore
+              } else if (request.queryParamsValues(param).length > 1) {
```

Comment:
maybe pull out this into its own variable on top:

```var paramsValue = request.queryParamsValues(param)```

and then use that variable to check the length / create list. makes it a bit more readable